packages/perseus/src/widgets/interactive-graphs/mafs-styles.css

```diff
@@ -152,6 +152,17 @@
     cursor: grab;
 }
 
+.MafsView .movable-circle .circle {
+    stroke: var(--mafs-blue);
```

Comment:
Question: Was the color here decided in the design document or is that just what we're using for mafs graphs? Not WB color tokens?

Reply:
We've been using this `--mafs-blue` variable in the CSS for Mafs components. It's the same as WB blue. We can't use WB tokens directly in CSS because (AFAIK?) WB only exports them as JavaScript.

Reply:
Thanks! Good to know it's the same as WB blue :)